docs/guides/launch.md

```diff
@@ -57,287 +57,83 @@ access it at `http://127.0.0.1:3000/` under credentials `admin/admin`.
 ## (Re)deploy db and contracts
 
 ```
-zk contract redeploy
+zkstack dev contracts
 ```
 
-## Environment configurations
-
-Env config files are held in `etc/env/target/`
-
-List configurations:
-
-```
-zk env
-```
+## Ecosystem Configuration
 
-Switch between configurations:
+Ecosystem configuration can be found in:
 
-```
-zk env <ENV_NAME>
-```
+- `/ZkStack.yaml`
+- `/configs` folder
+- `/chains` folder
 
-Default configuration is `dev.env`, which is generated automatically from `dev.env.example` during `zk init` command
-execution.
+These files are created at ecosystem initialization `zkstack ecosystem init` and at chain initialization
+`zkstack chain init`.
 
 ## Build and run server
 
 Run server:
 
+```bash
+zkstack server
 ```
-zk server
-```
-
-Server is configured using env files in `./etc/env` directory. After the first initialization, file
-`./etc/env/target/dev.env`will be created. By default, this file is copied from the `./etc/env/target/dev.env.example`
-template.
-
-Make sure you have environment variables set right, you can check it by running: `zk env`. You should see `* dev` in
-output.
 
 ## Running server using Google cloud storage object store instead of default In memory store
 
-Get the service_account.json file containing the GCP credentials from kubernetes secret for relevant environment(stage2/
-testnet2) add that file to the default location ~/gcloud/service_account.json or update object_store.toml with the file
-location
+TODO: check the new way to do this
```

Comment:
`service_account.json` part is the same
In order to run with GCS:
`zkstack prover init --bucket-base-url={url} --credentials-file={path/to/service_account.json}`

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/3134/commits/30f399101a1421fbfa4ac3994b463de9242a6762